framework/src/org/checkerframework/framework/util/QualifierPolymorphism.java

```diff
@@ -134,6 +134,34 @@ public QualifierPolymorphism(ProcessingEnvironment env, AnnotatedTypeFactory fac
         this.completer = new Completer();
     }
 
+    /**
+     * Creates a {@link QualifierPolymorphism} instance that uses the given checker for querying
+     * type qualifiers and the given factory for getting annotated types. Uses the specified {@code
+     * polyQuals} instead of finding them automatically.
+     *
+     * @param env the processing environment
+     * @param factory the factory for the current checker
+     * @param polyQuals mapping from top annotation to polymorphic annotation
+     */
+    protected QualifierPolymorphism(
```

Comment:
I had to add a new constructor to avoid the errors about more than one poly qualifier in the other constructor.